results/Mistral-7B-LoRA-RAHF-DUAL/alpaca_eval_gpt4/leaderboard.csv

```diff
@@ -0,0 +1,2 @@
+,win_rate,standard_error,mode,avg_length,n_wins,n_wins_base,n_draws,n_total,discrete_win_rate,length_controlled_winrate
```

Comment:
The leaderboard that you should add is in `src/alpaca_eval` not in `results`. Also note that you are running AlpacaEval 1 rather than AlpacaEval 2, is that on purpose?